tests/L0/run_transformer/run_bert_minimal_test.py

```diff
@@ -105,7 +108,7 @@ def train(model, optim, virtual_pipeline_model_parallel_size, pipeline_model_par
     hidden_size = global_vars.get_args().hidden_size
     forward_backward_func = get_forward_backward_func(virtual_pipeline_model_parallel_size, pipeline_model_parallel_size)
     tensor_shape = (args.seq_length, args.micro_batch_size, args.hidden_size)
-    for _ in range(8):
+    for i in range(16):
```

Comment:
Is `i` used anywhere?

```suggestion
    for _ in range(16):
```